pkg/workloads/tf_train/requirements.oob.txt

```diff
@@ -0,0 +1,11 @@
+numpy==1.15.4
+pandas==0.23.4
+scipy==1.2.0
+sympy==1.3
+statsmodels==0.9.0
+python-dateutil==2.7.5
+six==1.11.0
+wrapt==1.11.0
+requests==2.21.0
```

Comment:
These bottom three look strange, are they required by Cortex but listed here so the user can change the versions? Maybe we should add a comment here explaining